### ISSUE TITLE
[Question] Hosting Proivder functionality

### ISSUE THREAD

**Issue by ne0YT** [harvester/harvester#4697]
Is there a way to make custom users which can only create VMs in a certain namespace and with a limit on ressources / having his own backup maybe?

**Comment by noahgildersleeve** [comment]
You can do some of this with the Rancher integration. You can either set up a separate Rancher cluster or run it as a VM on Harvester.

For the resources limit you can take a look at [Resource Quotas](https://docs.harvesterhci.io/v1.2/rancher/resource-quota), and for the users you would want restrict it to the project with the resource quota. Something similar is outlined [here](https://harvester.github.io/tests/manual/_incoming/2394-2395-project-owner-customized-project-open-harvester/). 

Basically for RBAC (role based access control) we use upstream Rancher. So a lot of the specifics are outlined in the Rancher docs [here](https://ranchermanager.docs.rancher.com/pages-for-subheaders/manage-role-based-access-control-rbac).
